eng/pipelines/update-azure-linux-pipeline.yml

```
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# For info about runtime parameters, see https://github.com/microsoft/go-infra/blob/main/docs/pipeline-yml-style.md#runtime-parameters
+parameters:
+  # Allow disabling specific parts of the internal build release. These can be used to re-run
+  # specific parts of a release if necessary, e.g. if a step silently failed.
+  - name: runCreatePullRequest
+    displayName: Create Pull Request on Azure Linux repo
+    type: boolean
+    default: true
+
+  # This parameter intentionally has no default: see release-go-pipeline.yml
+  - name: goReleaseConfigVariableGroup
+    displayName: '[Use "go-release-config" for a real release] Variable group that specifies release target locations and auth.'
+    type: string
+
+  - name: pipelineBuildID
+    displayName: 'The ID of the microsoft-go build pipeline that triggered this pipeline.'
+    type: string
+    default: '2515771' # TODO: Remove default value
+trigger: none
```

Nit: keep newline (including default value removal while I'm here)

```suggestion

trigger: none
```

---

fixed